src/EnergyPlus/DataSurfaces.hh

```diff
@@ -1001,55 +1054,19 @@ namespace DataSurfaces {
         Real64 OutsideRevealSolAbs; // Solar absorptance of outside reveal
-        Real64 BmSolAbsdInsReveal;  // Multiplied by BeamSolarRad, gives beam solar absorbed
         // by inside reveal surfaces (m2)
-        Real64 BmSolRefldInsReveal; // Multiplied by BeamSolarRad, gives beam solar reflected
         // by inside reveal surfaces (m2)
-        Real64 BmSolRefldInsRevealReport;  // Beam solar reflected by inside reveal surfaces, for reporting (W)
-        Real64 BmSolRefldOutsRevealReport; // Beam solar reflected by outside reveal surfaces, for reporting (m2)
-        Real64 BmSolAbsdOutsReveal;        // Multiplied by BeamSolarRad, gives beam solar absorbed by
         // outside reveal surfaces (m2)
-        Real64 OutsRevealDiffOntoGlazing; // Multiplied by BeamSolarRad, gives diffuse from beam reflection from
         //    outside reveal that is incident on the glazing per m2 of glazing (-)
-        Real64 InsRevealDiffOntoGlazing; // Multiplied by BeamSolarRad, gives diffuse from beam reflection
-        //  from inside reveal that is incident on the glazing per m2 of glazing (-)
-        Real64 InsRevealDiffIntoZone; // Multiplied by BeamSolarRad, gives diffuse from beam reflection
-        //  from inside reveal that goes into zone directly or reflected from glazing (m2)
-        Real64 OutsRevealDiffOntoFrame; // Multiplied by BeamSolarRad, gives diffuse from beam reflection from outside reveal
-        //   that is incident on the outside of the frame per m2 of frame (-)
-        Real64 InsRevealDiffOntoFrame; // Multiplied by BeamSolarRad, gives diffuse from beam reflection from inside reveal
-        //   that is incident on the outside of the frame per m2 of frame (-)
-        // added for debugging CR 7596. TH 5/26/2009
-        Real64 InsRevealDiffOntoGlazingReport; // Diffuse solar from beam reflection
-        //  from inside reveal that is incident on the glazing (W)
-        Real64 InsRevealDiffIntoZoneReport; // Diffuse from beam reflection
-        //  from inside reveal that goes into zone directly or reflected from glazing (W)
-        Real64 InsRevealDiffOntoFrameReport; // Diffuse from beam reflection from inside reveal
-        //  that is incident on the frame (W)
-        Real64 BmSolAbsdInsRevealReport;        // Beam solar absorbed by inside reveal (W)
-        Real64 BlTsolBmBm;                      // Time-step value of blind beam-beam solar transmittance (-)
-        Real64 BlTsolBmDif;                     // Time-step value of blind beam-diffuse solar transmittance (-)
-        Real64 BlTsolDifDif;                    // Time-step value of blind diffuse-diffuse solar transmittance (-)
-        Real64 BlGlSysTsolBmBm;                 // Time-step value of blind/glass system beam-beam solar transmittance (-)
-        Real64 BlGlSysTsolDifDif;               // Time-step value of blind/glass system diffuse-diffuse solar transmittance (-)
+              // Beam solar absorbed by inside reveal (W)
```

Comment:
@XuanLuoLBL Some of the end-line comments that are broken into a second line are out of place. e.g. [here](https://github.com/NREL/EnergyPlus/blob/5a67bb5a6dd27d9f36c8536e7a96fd001e6d84c8/src/EnergyPlus/DataSurfaces.hh#L1064-L1068)